src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const clear = () => {
+    for (const param in state) {
+      delete state[param];
+    }
+  };
+
+  const addProps = (props) => {
+    Object.assign(state, props);
+  };
+
+  const removeProps = (keys) => {
+    for (const key of keys) {
+      delete state[key];
+    }
+  };
```

Comment:
let's move it to outside of this function (now u create those functions on each call of transformState)